lib/sign-up.js

```diff
@@ -23,6 +23,8 @@ function signUp (state, options) {
     return Promise.reject(new Error('SignUp with profile data not yet implemented. Please see https://github.com/hoodiehq/hoodie-account-client/issues/11.'))
   }
 
+  options.createdAt = new Date().toISOString()
```

Comment:
thanks for the explanation, let me just re-iterate to make sure I'm getting this

## high level
Is the purpose of implementing `createdAt` to check how long a user has been using the app anonymously before signing up? [hoodie/hoodie#282](https://github.com/hoodiehq/hoodiehq/issues/282) Looking at the difference in createdAt to the store signedUpAt, you'll render a different view?
## implementation
Looking at the suggestion, If the user doesn't exist then [storedAccount](https://github.com/hoodiehq/hoodie-account-client/blob/master/utils/get-state.js#L24) would return null. Would I rewrite the value, e.g. `storedAccount = {createdAt: new Date().toISOString()}`? Or should I add the `createdAt` [here](https://github.com/hoodiehq/hoodie-account-client/blob/master/utils/get-state.js#L44)?

I might be misinterpreting what [storedAccount](https://github.com/hoodiehq/hoodie-account-client/blob/master/utils/get-state.js#L24) would return. From my current understanding it looks like if a User exists then it should already have the `createdAt` attached to the account? Or is it possible for an existing user not to have `createdAt` attached?

Sorry if my explanation is unclear. I'll try my best to further explain any parts that sound confusing.